### ISSUE TITLE
CMake install has an extra include directory

### ISSUE THREAD

**Issue by prince-chrismc** [beached/daw_json_link#157]
Currently the project is installed in `/usr/local/include/include/daw/...`

The error comes from,

https://github.com/beached/daw_json_link/blob/bc332512277558762cb3a441faf711fa41ab3189/CMakeLists.txt#L104

When reading the docs... https://cmake.org/cmake/help/v3.13/command/install.html#directory

>The directory structure is copied verbatim to the destination. The last component of each directory name is appended to the destination directory but a trailing slash may be used to avoid this because it leaves the last component empty.

CMake provides the example:

```
install(DIRECTORY icons scripts/ DESTINATION share/myproj
        [...])
```

> will install the icons directory to share/myproj/icons and the scripts directory to share/myproj.

The code in questions adds an include folder to the include destination (this is a very confusing CMake feature :confused: )

**Comment by prince-chrismc** [comment]
Might be related to #147

**Comment by beached** [comment]
Thanks for pointing that out.  Should be fixed as of release v2.0.2
